examples/models/llama2/llama_transformer.py

```diff
@@ -260,21 +260,22 @@ class Attention(nn.Module):
     def __init__(self, args: ModelArgs, layer_id: int):
         super().__init__()
         self.use_kv_cache = args.use_kv_cache
-        self.n_kv_heads = args.n_heads if args.n_kv_heads is None else args.n_kv_heads
-        assert args.n_heads % self.n_kv_heads == 0
+        self.n_heads = args.n_heads
```

Comment:
I guess it's still draft, but will be good to have a seperate PR if we need to change `llama_transformer.py`..

Reply:
Sure. We can separate this part to another PR.

Reply:
Hi @cccclai 
Just pushed [a seperated PR](https://github.com/pytorch/executorch/pull/6376) for it. If PR 6376 looks fine to you and is merged, I will rebase this PR. Thank you. :)